swap/tests/bob_refunds_using_cancel_and_refund_command.rs

```diff
@@ -1,17 +1,20 @@
 pub mod testutils;
 
-use swap::protocol::bob;
 use swap::protocol::bob::BobState;
+use swap::protocol::{alice, bob};
 use testutils::bob_run_until::is_btc_locked;
 use testutils::FastCancelConfig;
 
 #[tokio::test]
 async fn given_bob_manually_refunds_after_btc_locked_bob_refunds() {
     testutils::setup_test(FastCancelConfig, |mut ctx| async move {
-        let (bob_swap, bob_join_handle) = ctx.new_swap_as_bob().await;
+        let (bob_swap, bob_join_handle) = ctx.bob_swap().await;
+        let bob_swap = tokio::spawn(bob::run_until(bob_swap, is_btc_locked));
 
-        let bob_state = bob::run_until(bob_swap, is_btc_locked).await.unwrap();
+        let alice_swap = ctx.alice_swap_triggered_by_bob().await;
+        let _alice_swap = tokio::spawn(alice::run(alice_swap));
```

Comment:
If you don't care about the handle, I would just not assign it a variable. It will not emit a warning :)


```suggestion
        tokio::spawn(alice::run(alice_swap));
```

Reply:
Yeah, in this case `_` is OK - I often like to to use `_purpose` to express what we are "ignoring" - find it more readable